dashboard/src/components/ui/v2/ListItem/ListItemText.tsx

```diff
@@ -14,25 +21,35 @@ const StyledListItemText = styled(MaterialListItemText)(({ theme }) => ({
   gap: theme.spacing(0.25),
   fontSize: theme.typography.pxToRem(15),
   [`&.${listItemTextClasses.root}`]: {
-    margin: 0,
+    margin: 0
+  },
+  [`&.${listItemTextClasses.warning}`]: {
+    color: theme.palette.warning.main
```

Comment:
```suggestion
    color: theme.palette.warning.dark,
```

I would go with the darker color to make the text easier to read.